Frameworks/UIKit/UIView.mm

```diff
@@ -198,27 +199,68 @@ - (bool)_processGesturesForTouch:(UITouch*)touch event:(UIEvent*)event touchEven
         }
     }
 
-    // gesture priority list
-    const static id s_gesturesPriority[] = {[UIPinchGestureRecognizer class],
-                                            [UISwipeGestureRecognizer class],
-                                            [UIPanGestureRecognizer class],
-                                            [UILongPressGestureRecognizer class],
-                                            [UITapGestureRecognizer class] };
 
-    const static int s_numGestureTypes = sizeof(s_gesturesPriority) / sizeof(s_gesturesPriority[0]);
+    //
+    // Remove any gestures that should not simulatanously be recognized
+    //
 
-    //  Process all gestures
-    for (int i = 0; i < s_numGestureTypes; i++) {
-        id curgestureClass = s_gesturesPriority[i];
-        id gestures = [g_curGesturesDict objectForKey:curgestureClass];
-        if ([curgestureClass _fireGestures:gestures]) {
-            handled = true;
-            if (handled && DEBUG_GESTURES) {
-                TraceVerbose(TAG, L"Gesture (%hs) handled.", object_getClassName(curgestureClass));
+    // First find the active gesture
+    UIGestureRecognizer* activeGesture = nil;
+
+    int count = [g_currentlyTrackingGesturesList count];
+    for (int i = count - 1; i >= 0; i--) {
+        UIGestureRecognizer* curGesture = [g_currentlyTrackingGesturesList objectAtIndex:i];
+        UIGestureRecognizerState state = curGesture.state;
+        if ( state != UIGestureRecognizerStatePossible && state != UIGestureRecognizerStateCancelled) {
+            activeGesture = curGesture;
+            break;
+        }
+    }
+
+    if ( activeGesture ) {
```

Comment:
Strange; it *should have* modified the file in place to remove the spacing inconsistencies.  it might be worth trying .\tools\scripts\clang-format.ps1 -File <path to file> to see if you get different results?

---
In reply to: [78604452](https://github.com/Microsoft/WinObjC/pull/1002#discussion_r78604452) [](ancestors = 78604452)